.travis.yml

```diff
@@ -29,18 +34,22 @@ before_install:
   - ./miniconda.sh -b -p ./miniconda
   - export PATH=`pwd`/miniconda/bin:$PATH
   - conda update --yes conda
-  - conda config --append channels conda-forge
-  - conda create -y -q -n test-env python=$TRAVIS_PYTHON_VERSION
+  - conda config --add channels conda-forge --force
+  - conda config --set channel_priority strict
+  - conda create -y --name test-env python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements_tests.txt
   - source activate test-env
 
 install:
   - conda install --yes pip
   - if "$PYSAL_PYPI"; then
         echo 'testing pypi libpysal' && pip install libpysal;
-        else echo 'testing git libpysal'; git clone https://github.com/pysal/libpysal.git; cd libpysal; pip install .; cd ../;
+    else
+        echo 'testing git libpysal';
+        pip install https://github.com/pysal/libpysal/archive/master.zip;
```

Comment:
maybe this line is the culprit? are we grabbing from the archived master here? [this is the corresponding line for `spaghetti`](https://github.com/pysal/spaghetti/blob/68b536f387f8c0ce5e70566b6aaa52ff49b3d2b4/.travis.yml#L64)

Reply:
It should be fine - `splot` and `giddy` are using the same strategy. https://github.com/pysal/libpysal/archive/master.zip grabs the current version of libpysal on the `master` branch.

Reply:
hmmm. I will try to make some time to give this attention in the next few days.